app/views/anonymous_feedback/explore/new.html.erb

```diff
@@ -12,19 +12,41 @@
 <div class="row">
   <div class="col-md-8">
     <h2 class="add-bottom-margin">By URL</h2>
-    <%= semantic_form_for @explore_by_url, url: { action: "create" }, html: {class: 'well'} do |f| %>
-      <%= f.input :url, label: "URL", required: true, input_html: { class: "input-md-7", :"aria-required" => true, type: "url"} %>
-      <p class="help-block add-bottom-margin">For example a page like <code>/vat-rates</code> or <code>https://www.gov.uk/bank-holidays</code>, or a partial path like <code>/guidance/</code></p>
+    <%= semantic_form_for @explore_by_multiple_paths, url: { action: "create" }, html: {class: 'well'} do |f| %>
+      <%= f.input :list_of_urls, label: "URL(s)", required: true, input_html: { class: "input-md-7", :"aria-required" => true } %>
+      <p class="help-block add-bottom-margin">
+        For example a page like <code>/vat-rates</code> or <code>https://www.gov.uk/bank-holidays</code>, or a partial path like <code>/guidance/</code>
+        <br/>
+        You can also search by a comma separated list: <code>/vat-rates, https://www.gov.uk/bank-holidays, /guidance/</code>
+        <br/>
+        For longer lists, please upload a file with the comma separated list (see below in "Upload a URL List").
+      </p>
       <%= f.action :submit, label: "Explore by URL", button_html: { class: "btn btn-success" } %>
     <% end %>
   </div>
+
+  <div class="col-md-8">
+    <h2 class="add-bottom-margin">Upload a URL List</h2>
+    <%= semantic_form_for @explore_by_multiple_paths, url: { action: "create" }, html: { class: 'well', multipart: true } do |f| %>
+      <%= f.inputs do %>
+        <%= f.input :uploaded_list, as: :file, label: "Upload list of URLs", required: true, input_html: { class: "input-md-7", :"aria-required" => true, type: "file"} %>
+      <% end %>
+      <p class="help-block add-bottom-margin">
+        The contents of the file should be a comma separated list:
+        <code>/vat-rates, https://www.gov.uk/bank-holidays, /guidance/</code>
+      </p>
+      <%= f.action :submit, label: "Upload list of urls", data: { confirm: "This will email you a CSV export. Are you sure you want to continue?" }, button_html: { class: "btn btn-success" } %>
```

Comment:
Doesn't this go to the browse page now? We can drop the "This will emails you a CSV export" warning.

Reply:
Dropped